### PR TITLE
Fix IOOBE caused by jxBrowser isPageLoading() method.

### LINKED ISSUE #291: Multiple person cards selected if card is dragged quickly
Steps to reproduce:
- Click on the bottom-most card
- Quickly drag the top-most card down at least by 1 card  
Result: 2 cards are selected, `IndexOutOfBoundsException` is thrown

This should work with any card, just indicated top/bottom-most for simplicity.

### PR DESCRIPTION
**Pull request by yl-coder**
Link to #331 
Fixes #291 

The purpose of this fix is to avoid the usage of isPageLoading() incase you find the changes odd.

### CODE REVIEW

**Comment by damithc** [comment]
Need to isolate the problem, figure out the steps to reproduce, and report to jxBrowser.

**Comment by yl-coder** [comment]
Yes, this will be a hotfix for now. Already created an issue for the problem.

**Comment by yl-coder** [comment]
@HansNewbie Can help to review this? This is to fix the issue that we were talking about on tueday evening.

File: src/main/java/address/browser/BrowserManager.java [hunk 1]
```diff
@@ -123,6 +122,12 @@ public synchronized void loadProfilePage(ReadOnlyViewablePerson person) {
         selectedPersonUsername.addListener(listener);
     }
 
+    private void configureGithubPageTasks(List<Page> pages) {
+        List<GithubProfilePage> githubProfilePages = pages.stream().map(p
+                -> new GithubProfilePage(p)).collect(Collectors.toCollection(ArrayList::new));
+        githubProfilePages.stream().forEach(GithubProfilePage::setupPageAutomation);
+    }
```
Comment (HansNewbie) [inline src/main/java/address/browser/BrowserManager.java]: - we can combine the two so no need for the list.
- we can use method reference for the creation of github profile page.

`pages.stream().map(GithubProfilePage::new).forEach(GithubProfilePage::setupPageAutomation);`

File: src/main/java/hubturbo/embeddedbrowser/HyperBrowser.java [hunk 2]
```diff
@@ -159,9 +159,12 @@ public synchronized Page loadUrls(URL url, List<URL> futureUrl) throws IllegalAr
         Page page = loadPage(url);
         replaceBrowserView(page.getBrowser().getBrowserView());
         displayedUrl = url;
-        futureUrl.forEach(this::loadPage);
 
-        return page;
+        List<Page> pages = new ArrayList<>();
+        pages.add(page);
+        futureUrl.forEach(p -> pages.add(this.loadPage(p)));
```
Comment (HansNewbie) [inline src/main/java/hubturbo/embeddedbrowser/HyperBrowser.java]: Should be `futureUrls` :p

File: src/main/java/hubturbo/embeddedbrowser/jxbrowser/JxBrowserAdapter.java [hunk 3]
```diff
@@ -105,7 +105,7 @@ public void reset() {
         this.browserView.getBrowser()
                         .getLoadListeners()
                         .stream().forEach(listener -> browserView.getBrowser().removeLoadListener(listener));
-        this.browserView.sceneProperty().removeListener(sceneChangeListener);
+        this.attachListener = Optional.empty();
```
Comment (HansNewbie) [inline src/main/java/hubturbo/embeddedbrowser/jxbrowser/JxBrowserAdapter.java]: What's this for? Seems to be not related to PR?

File: src/main/java/address/browser/BrowserManager.java (hunk 1 above)
Comment (yl-coder) [inline src/main/java/address/browser/BrowserManager.java]: ok right.

File: src/main/java/hubturbo/embeddedbrowser/HyperBrowser.java (hunk 2 above)
Comment (yl-coder) [inline src/main/java/hubturbo/embeddedbrowser/HyperBrowser.java]: ok :p

File: src/main/java/hubturbo/embeddedbrowser/jxbrowser/JxBrowserAdapter.java (hunk 3 above)
Comment (yl-coder) [inline src/main/java/hubturbo/embeddedbrowser/jxbrowser/JxBrowserAdapter.java]: The previous is a mistake. attachListener is the listener that new page passed in.

**Comment by HansNewbie** [comment]
No more comments.

**Pull request merged**